src/Microsoft.AspNetCore.Routing/Internal/DispatcherMarkerService.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Routing.Internal
+{
+    /// <summary>
+    /// A marker class used to determine if all the dispatcher services were added
+    /// to the <see cref="IServiceCollection"/> before dispatcher is configured.
+    /// </summary>
+    public class DispatcherMarkerService
```

Comment:
Does this have to be pubternal? routing is like, 1 assembly?

Reply:
I don't think so but I followed the pattern of the existing check for routing and RouteMarkerService was pub internal 🤷‍♂️

Reply:
Yeah, that was written years ago, I'd suggest just making it internal. 

If you that makes you sad that it's inconsistent feel free to change `RouteMarkerService`.

Reply:
Changed both